tox.ini

```
@@ -125,6 +125,7 @@ envlist =
 # but these tests still work.
     pymongo_contrib-py38-pymongo{30,31,32,33,35,36,37,38,39,310,}-mongoengine
     pymysql_contrib-py{27,35,36,37,38}-pymysql{07,08,09,}
+    pynamodb_contrib-{py27,py35,py36,py37,py38}-pynamodb{43}
```

```suggestion
    pynamodb_contrib-{py27,py35,py36,py37,py38}-pynamodb{43,}
```

this will run the test with the latest version of pynamodb